data/tlg2035/tlg003/tlg2035.tlg003.opp-lat1.xml

```diff
@@ -100,7 +108,7 @@
 </revisionDesc>
 </teiHeader>
   <text>
-    <body><div type="edition" n="urn:cts:greekLit:tlg2768.tlg002.opp-lat1" xml:lang="lat">
+    <body><div type="edition" n="urn:cts:greekLit:tlg2035.tlg003.opp-lat1" xml:lang="lat">
```

Comment:
As with in the metadata file, this should be `<div type="translation">` instead of `type="edition"`.